mopidy/softwaremixer/mixer.py

```diff
@@ -36,15 +43,47 @@ def teardown(self):
     def get_volume(self):
         if self._audio_mixer is None:
             return None
-        return self._audio_mixer.get_volume().get()
+        mixer_volume = self._audio_mixer.get_volume().get()
+        if mixer_volume == round(self._logical_volume):
+            mixer_volume = self._logical_volume
```

Comment:
Would you be able to add a comment to the code explaining what's going on here?

Reply:
Also a good point. I did this in a different place, but forgot to on the other end of the process. I've now included a detailed comment with an example case showing why it matters.